packages/core/src/components/menu/_common.scss

```diff
@@ -14,9 +14,19 @@ $menu-item-padding-large: ($pt-button-height-large - $pt-icon-size-large) / 2 !d
 $menu-background-color: $white !default;
 $dark-menu-background-color: $dark-gray4 !default;
 
+$menu-item-color-hover: $minimal-button-background-color-hover;
+$menu-item-color-active: $minimal-button-background-color-active;
+$dark-menu-item-color-hover: $dark-minimal-button-background-color-hover;
+$dark-menu-item-color-active: $dark-minimal-button-background-color-active;
+
 // customize modifier classes with params.
 // setting modifier to "" will generally apply it as default styles due to & selectors
-@mixin menu-item($disabled-selector: ".pt-disabled", $hover-selector: ":hover") {
+@mixin menu-item(
+  $active-selector: ".pt-active",
+  $disabled-selector: ".pt-disabled",
+  $active-pseudoselector: ":active",
+  $hover-pseudoselector: ":hover"
```

Comment:
The pseudoselectors were merely used as variable references, I agree that doesn't warrant them as params.

Because sass doesnt support `null` params, I don't think there's a way to "not support part of the mixin" (it just falls through as in the comment in the code) - except from setting the selector to something like `.unstyled`. The same goes for `.pt-active` and `.pt-disabled`. If we're anticipating that usage, either that or we could split up the mixin.

`$active-selector: "&:active, &.pt-active"` won't work because we do different things with them